osu.Game/Screens/OnlinePlay/Lounge/Components/PlaylistCountPill.cs

```diff
@@ -41,15 +41,22 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            Playlist.BindCollectionChanged(updateCount, true);
+            PlaylistItemStats.BindValueChanged(_ => updateCount(), true);
+            Playlist.BindCollectionChanged((_, __) => updateCount(), true);
```

Comment:
nit: don't need to run twice

```suggestion
            PlaylistItemStats.BindValueChanged(_ => updateCount());
            Playlist.BindCollectionChanged((_, __) => updateCount(), true);
```